kong/templates/nginx.lua

```diff
@@ -23,17 +23,44 @@ events {
 > end
 }
 
-> if wasm and wasm_modules_parsed and #wasm_modules_parsed > 0 then
+> if wasm then
+load_module $(wasm_dynamic_module);
+
 wasm {
+  env RUST_BACKTRACE=1;
+  env WASMTIME_BACKTRACE_DETAILS=1;
```

Comment:
It looks like this PR adds the ability to define wasm shm_kv memory zones, so it's probably as good of a time as any to remove `shm_kv kong_wasm_rate_limiting_counters 12m;` from this template and the one in ./spec/fixtures.

Reply:
related: https://github.com/Kong/kong/pull/11233